apps/pie-docs/src/all-about-pie/roadmap.md

```diff
@@ -14,7 +14,7 @@ description: This roadmap displays our current projects and future plans. Its ai
   * Navigation enhancements
 * Identifying and fixing accessibility violations
 * Supporting internal JET priorities
-* Build out a proof-of-concept (PoC) for a native Web Component implementation of our PIE component system using the [Lit Web Component library](https://lit.dev/).
+* Build out a proof-of-concept (PoC) for a native Web Component implementation of our PIE component system using the [Lit Web Component library](https://lit.dev/)
```

Comment:
should this be `building` if we're changing all the bullet points to use `ing` words 😄

Reply:
Nice spot @LTurns just updating this now :)